.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -20,21 +20,21 @@ assignees: ''
 
         * https://github.com/confidential-containers/image-rs/blob/main/Cargo.toml
           * Change the revision
-          * Run `cargo update -p image-rs`
+          * Run `cargo update -p attestation_agent`
```

Comment:
Oops, my bad.

While checking this (which is something I should've done earlier ...) I've noticed:
```
⋊> image-rs on main ≡ ⨯ cargo update -p attestation_agent
error: There are multiple `attestation_agent` packages in your project, and the specification `attestation_agent` is ambiguous.
Please re-run this command with `-p <spec>` where `<spec>` is one of the following:
  https://github.com/confidential-containers/attestation-agent#attestation_agent@0.1.0
  https://github.com/confidential-containers/attestation-agent.git#attestation_agent@0.1.0
```

Would it be worth to just add the full resulting command?
```
cargo update -p https://github.com/confidential-containers/attestation-agent.git#attestation_agent@0.1.0
```

Reply:
does that work for you? for me the output is:
```
$ cargo update -p attestation_agent
error: There are multiple `attestation_agent` packages in your project, and the specification `attestation_agent@0.1.0` is ambiguous.
Please re-run this command with `-p <spec>` where `<spec>` is one of the following:
  https://github.com/confidential-containers/attestation-agent#attestation_agent@0.1.0
  https://github.com/confidential-containers/attestation-agent#attestation_agent@0.1.0
```
and rerunning has the same output.

This appears to happen if Cargo.lock is present, removing it seems to allow the first update to work.